src/material/autocomplete/_autocomplete-theme.scss

```diff
@@ -75,6 +76,8 @@
 }
 
 @mixin _theme-from-tokens($tokens) {
+  @include validation.selector-defined(
+      'Calls to Angular Material theme mixins with an M3 theme must be wrapped in a selector');
```

Comment:
Without reading material-3.md, it's not obvious to me why this error message is popping up and what action to take.

Maybe something to this effect:
"Autocomplete mixin called without a selector. This can cause unspecified behavior, and Angular Material requires any `@include` of the `-theme`, `-color`, `-typography`, `-density`, or `-base`
  mixins to be wrapped in a selector. Call theme mixin inside a selector."

Reply:
Many people have wrapper mixins, or may even be using our own wrapper mixin : `mat.all-component-themes`, so the name of this mixin may not be the call they actually need to wrap. The stack trace should be able to point them to the right place anyways.